apps/vc-api/src/main.ts

```diff
@@ -19,11 +19,15 @@ import { setupApp, setupSwaggerDocument } from './setup';
 import { readFile } from 'fs/promises';
 import { resolve as resolvePath } from 'path';
 import { SwaggerModule } from '@nestjs/swagger';
+import { SeederService } from './seeder/seeder.service';
 
 async function bootstrap() {
   const app = await setupApp();
   SwaggerModule.setup('api', app, setupSwaggerDocument(app));
   await app.listen(3000);
+
+  await app.get(SeederService).seed();
```

Comment:
Maybe explaining why we are seeding could be useful in the future, though it risks getting out of date.
```suggestion
  // Seeding needed in order to be able to execute VC-API test suite
  // https://github.com/w3c-ccg/vc-api-test-suite
  await app.get(SeederService).seed();
```